STAN.CLIENT/pb/protocol.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015-2018 The NATS Authors
+// Copyright 2016-2018 The NATS Authors
```

Comment:
This should be reverted back to the copy on master.

Reply:
reverted to old proto